### DISCUSSION TITLE
why rerender when url change

### DISCUSSION THREAD

**Discussion by eightHundreds** [apache/skywalking#9055] [General]
https://github.com/apache/skywalking-booster-ui/blob/784c2e97b820a31cc3a13cb36dda3d0f6c038d8d/src/App.vue#L16

Is there no performance problem?

**Comment by kezhenxu94** [comment]
@Fine0830 I also noticed this when I select an item in the left sidebar, the entire page is re-rendered, any reason why we should do this, or can we improve this?

https://user-images.githubusercontent.com/15965696/168202815-d026cce8-1f73-4155-9c64-1ca31abb40dc.mov

**Reply by wu-sheng** [reply]
I think the page context is relative to context's layer, selected service/instance, etc. Waiting for the expert's :P

**Reply by Fine0830** [reply]
Uhhh...my fault. It's a bug. I will fix it. Or could @eightHundreds fix this problem?

**Reply by wu-sheng** [reply]
Fixed through https://github.com/apache/skywalking-booster-ui/pull/87
